python/doc/theory/data_analysis/chi2_fitting_test.rst

```diff
@@ -1,62 +1,63 @@
 .. _chi2_fitting_test:
 
-Chi-squared goodness of fit test
---------------------------------
-
-This method deals with the modelling of a probability distribution of a
-random vector :math:`\vect{X} = \left( X^1,\ldots,X^{n_X} \right)`. It
-seeks to verify the compatibility between a sample of data
-:math:`\left\{ \vect{x}_1,\vect{x}_2,\ldots,\vect{x}_N \right\}` and a
-candidate probability distribution previous chosen. The use of the
-:math:`\chi^2` Goodness-of-Fit test allows one to answer this
-question in the one dimensional case :math:`n_X =1`, and with a discrete
-distribution.
-
-Let us limit the case to :math:`n_X = 1`. Thus we denote
-:math:`\vect{X} = X^1 = X`. We also note that as we are considering
-discrete distributions i.e. those for which the possible values of
-:math:`X` belong to a discrete set :math:`\cE`, the candidate
-distribution is characterized by the probabilities
-:math:`\left\{ p(x;\vect{\theta}) \right\}_{x \in \cE}`.
-
-| The chi squared test is based on the fact that if the candidate
-  distribution is appropriate, the number of values in the sample x1,
-  x2, ..., xN that are equal to :math:`x` should be on average equal to
-  :math:`N p(x;\vect{\theta})`. The idea is therefore to compare the
-  “theoretical values” with the actual observed values. This comparison
-  is performed with the aid of the following “distance”.
+Chi-squared test
+----------------
+
+The :math:`\chi^2` test is a statistical test of whether a given sample of data is drawn
+from a given discrete distribution. The library only provides the :math:`\chi^2` test for
+distributions of dimension 1.
+
+We denote by :math:`\left\{ x_1,\dots,x_{\sampleSize} \right\}` a sample of dimension 1.
+Let :math:`F` be  the (unknown) cumulative distribution function of the discrete distribution.
+
```

Comment:
This blank line seems unnecessary to me, since the next sentence continues the flow of the previous one.
```suggestion
```